README.md

```diff
@@ -233,22 +233,32 @@ This is encountered when Robolectric has problems downloading the jars it needs
 ## Creating signed releases for Google Play Store
 Maintainers keep a folder with a clean checkout of the code and use [jenv.be](https://www.jenv.be) in that folder to ensure compilation with Java 11.
 
-Maintainers have a `local.properties` file in the root folder with the following:
-```
-sdk.dir=/path/to/android/sdk
-```
-
-Maintainers have a `secrets.properties` file in the `collect_app` folder with the following:
-```
-// collect_app/secrets.properties
-RELEASE_STORE_FILE=/path/to/collect.keystore
-RELEASE_STORE_PASSWORD=secure-store-password
-RELEASE_KEY_ALIAS=key-alias
-RELEASE_KEY_PASSWORD=secure-alias-password
-```
-
-Maintainers also have a `google-services.json` file in the `collect_app/src/odkCollectRelease` folder. The contents of the file are similar to the contents of `collect_app/src/google-services.json`.
-
-When ready to generate a build for the Play Store, maintainers tag the build by [adding a release](releases). Tags for full releases must have the format `vX.X.X`. Tags for beta releases must have the format `vX.X.X-beta.X`.
-
-To generate official signed releases, you'll need the keystore file, the keystore passwords, a configured `collect_app/secrets.properties` file, and a configured `collect_app/src/odkCollectRelease/google-services.json` file. Then run `./gradlew assembleOdkCollectRelease`. If successful, a signed release will be at `collect_app/build/outputs/apk`.
+### Release prerequisites:
+
+- a`local.properties` file in the root folder with the following:
+  ```
+  sdk.dir=/path/to/android/sdk
+  ```
+
+- the keystore file and passwords
+
+- a `secrets.properties` file in the `collect_app` folder with the following:
+  ```
+  // collect_app/secrets.properties
+  RELEASE_STORE_FILE=/path/to/collect.keystore
+  RELEASE_STORE_PASSWORD=secure-store-password
+  RELEASE_KEY_ALIAS=key-alias
+  RELEASE_KEY_PASSWORD=secure-alias-password
+  ```
+
+- a `google-services.json` file in the `collect_app/src/odkCollectRelease` folder. The contents of the file are similar to the contents of `collect_app/src/google-services.json`.
+
+### Release checklist:
+- tag the build by [adding a release](releases).
```

Comment:
This link doesn't seem to work

Reply:
It's a relative link that would work once the PR is merged. But I guess in this case there's only one set of releases so I've changed it to an absolute link.

Reply:
> It's a relative link that would work once the PR is merged.

But the one we already have in https://github.com/getodk/collect doesn't work as well.

Reply:
Oh, well then! Thanks for the fix.